README.md

```diff
@@ -30,112 +38,85 @@ Run Lightning Terminal with a local `lnd` instance:
 
-To upgrade from v0.1.1-alpha or earlier simply create a `lit.conf` file in your LiT directory. The default location LiT uses depends on your operating system:
+If you used command line arguments with previous versions then you don't need to
+change anything when upgrading.
+
+To upgrade from v0.1.1-alpha or earlier simply create a `lit.conf` file in your
+LiT directory. The default location LiT uses depends on your operating system:
+
 - MacOS: `~/Library/Application Support/Lit/lit.conf`
 - Linux: `~/.lit/lit.conf`
 - Windows: `~/AppData/Roaming/Lit/lit.conf`
 
-Move all the configuration settings specific to LiT from `lnd.conf` to `lit.conf` and remove any previous LiT-specific customizations from the configuration settings in `lnd.conf`. Note that any section headers (`[ Example ]`) in `lit.conf` should be removed or changed to comments (`# Example`). 
+Move all the configuration settings specific to LiT from `lnd.conf` to
+`lit.conf` and remove any previous LiT-specific customizations from the
+configuration settings in `lnd.conf`. Note that any section headers (
+`[ Example ]`) in `lit.conf` should be removed or changed to comments (
+`# Example`).
 
 ## Usage
-Read the [walkthrough](docs/WALKTHROUGH.md) document to learn more about how to use LiT.
+
+Read the [walkthrough](docs/WALKTHROUGH.md) document to learn more about how to
+use LiT.
 
 ## Troubleshooting
-If you encounter any issues please see our [troubleshooting guide](./docs/troubleshooting.md).
+
+If you encounter any issues please see
+our [troubleshooting guide](./docs/troubleshooting.md).
 
 ## Build from source
-If you’d prefer to compile from source code please follow [these instructions](./docs/compile.md).
+
+If you’d prefer to compile from source code please
+follow [these instructions](./docs/compile.md).
 
 ## Compatibility
 
 Full Lightning Terminal functionality can be dependent on running a compatible
-version of `lnd`. View the table below to ensure that you run a compatible
-version of `lnd` with the relevant `litd` release.
-
-This version compatibility dependency table is only relevant when running litd
-in remote mode (meaning that `lnd-mode=remote` is set). It shows the
-**_minimum_** `lnd` version that can be used with different versions of `litd`
-(LiT). The bundled version will always come with the correct,
+version of `lnd`. For each LiT release, the minimum compatible `lnd` version
+should be specified in the [GitHub release
+notes](https://github.com/lightninglabs/lightning-terminal/releases). If a
+release note does not list it (for example, older releases), see the archived
+compatibility table in [docs/compatibility.md](./docs/compatibility.md).
+
+This compatibility requirement only applies when running LiT in remote mode
+(`lnd-mode=remote`). The bundled version will always come with the correct,
 [compatible versioning](#daemon-versions-packaged-with-lit).
```

Comment:
We should drop the "[compatible versioning](#daemon-versions-packaged-with-lit)" part of this here.